Public/Src/Tools/SymbolDaemon/Program.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Net;
+using BuildXL.Ipc;
+using BuildXL.Ipc.Interfaces;
+using BuildXL.Utilities.CLI;
+using Tool.ServicePipDaemon;
+using static Tool.ServicePipDaemon.Statics;
+
+namespace Tool.SymbolDaemon
+{
+    /// <summary>
+    /// SymbolDaemon entry point.
+    /// </summary>
+    public static class Program
+    {
+        /// <nodoc/>        
+        public static int Main(string[] args)
+        {
+            // TODO:#1208464 - this can be removed once SymbolDaemon targets .net 4.7 or newer where TLS 1.2 is enabled by default
+            ServicePointManager.SecurityProtocol = ServicePointManager.SecurityProtocol | SecurityProtocolType.Tls12;
+
+            try
+            {
+                Console.WriteLine(nameof(SymbolDaemon) + " started at " + DateTime.UtcNow);
+                Console.WriteLine(SymbolDaemon.SymbolDLogPrefix + "Command line arguments: ");
+                Console.WriteLine(string.Join(Environment.NewLine + SymbolDaemon.SymbolDLogPrefix, args));
+                Console.WriteLine();
+
+                SymbolDaemon.EnsureCommandsInitialized();
+
+                var confCommand = ServicePipDaemon.ServicePipDaemon.ParseArgs(args, new UnixParser());
+                if (confCommand.Command.NeedsIpcClient)
+                {

```

Comment:
Could you please add a comment here, saying that this is never executed in production because Main method is initiated only by Service Process pips.